pkg/autodiscovery/providers/kube_services.go

```diff
@@ -115,6 +115,41 @@ func (k *KubeServiceConfigProvider) invalidateIfChanged(old, obj interface{}) {
 		k.upToDate = false
 		return
 	}
+	if valuesDiffer(castedObj.Annotations, castedOld.Annotations, kubeEndpointAnnotationPrefix) {
+		log.Trace("Invalidating configs on service change")
+		k.upToDate = false
+		return
+	}
+}
+
+func parseServiceAnnotations(services []*v1.Service) ([]integration.Config, error) {
```

Comment:
Could you please move the method back to its original location in the file so the diff is computed, instead of removing/adding the whole method?

Reply:
Done :)